test/unit/spec/OktaSignIn_spec.js

```diff
@@ -386,4 +411,45 @@ Expect.describe('OktaSignIn v2 bootstrap', function () {
       });
     });
   });
-});
+
+  Expect.describe('Directly consumes proxyIdxResponse from widget config and do not introspect state token', function () {
```

Comment:
we cam get rid of these tests.

Reply:
I guess it's fine to leave a simple sanity check here ?